techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf

```diff
@@ -115,6 +115,7 @@ bundle agent propagate_promises(trackingkey, technique_name, relay_sync_method,
       "${g.shared_files}"
         copy_from    => remote("${server_info.policy_server}","${g.shared_files}"),
         depth_search => recurse_visible("inf"),
+        perms => mog("640", "root", "rudder"),
```

Comment:
not sure about this, we may need to preserver permissions instead

Reply:
I'l check how it could work before

Reply:
Apparently we didn't preserve perms there back in 3.1, so let's assume we don't need it.